client/components/dataviews/style.scss

```diff
@@ -215,3 +215,20 @@
 		padding-left: 48px;
 	}
 }
+
+// TODO: remove when this is implemented upstream.
+//
+// PR: https://github.com/Automattic/wp-calypso/pull/95561
```

Comment:
Thanks for doing this. No need to add this PR, the git blame would point to it anyway :) Though a little explanation as to _why_ there were needed wouldn't hurt. If I understood this correctly, it'd be something like

> This reverts to core styles that are overriden by wp-calypso (where?),
> which need to be removed in a follow-up.

Reply:
Nice, [added it here](https://github.com/Automattic/wp-calypso/pull/95561/commits/188f36f1435bf656cfdaf9f02dd6d57d002b36ce).